test/genomesets_and_multiples.pl

```diff
@@ -54,34 +54,54 @@ sub reannotate_genomes {
     return make_impl_call("RAST_SDK.annotate_genomes", $params);
 }
 
-my $DEBUG = 'Y';
-my $genome_obj_name1 = "Dactylopius coccus";
+my $DEBUG = 'N';
+my $genome_obj_name1 = "Dactylopius_coccus";
 my $genome_ref1 = "40046/2/1";
-my $genome_obj_name2 = "Drosophila melanogaster";
+my $genome_obj_name2 = "Drosophila_melanogaster";
 my $genome_ref2 = "40046/3/1";
+my $genome_obj_name3 = "Nomada_ferruginata";
+my $genome_ref3 = "40046/4/1";
```

Comment:
Some would object to hard coded refs in test but I understand that they are often the most practical way of geting things done. I would recommend that you note the environment for the refs and make sure that the refs are in a public WS for future testers

Reply:
@JamesJeffryes 
As I work on the tests, I'm trying to evolve them so I have a fast way to run when I'm debugging (e.g. "40046/4/1") and a way that they will run for other people (.e.g., "Nomada_ferruginata"). I haven't upgraded them all yet. This commit was to just get caught up with all the new tests that I have added as I have added new functionality. 

Do you want me to work your comments into this commit or work it into future work?  I don't to interrupt you anymore than needed before the GSP.